src/app/index.js

```diff
@@ -35,7 +35,7 @@ class App extends React.Component {
 
   render() {
     const { location, history, settings } = this.props;
-    const theme = settings.DarkMode ? 'dark' : 'light';
+    const theme = settings.darkMode ? 'dark' : 'light';
```

Comment:
💡 I think it would make it easier to handle the `isSigninFlow` here:
```suggestion
    const theme = settings.darkMode && !routeObj.isSigninFlow ? 'dark' : 'light';
```

It would mean:
- no need to pass `isSigninFlow` to `OfflineWrapper` and `theme` HOC
- no need for `noTheme` prop on `Icon` in `signInHeader.js`